src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
Consider increasing the specificity of the selectors for the active stars by including the `.stars__star` class in the selector. This will ensure that the styles are applied correctly even if there are other nested elements or styles that might interfere.